### ISSUE TITLE
Build with Docker failed

### ISSUE THREAD

**Issue by Snooowgh** [PlatONnetwork/PlatON-Go#1537]
#### System information

PlatON version: `0.13.1_alpha`
OS & Version: OSX

Build with Docker failed:
```cmd
docker build .
```
#### Backtrace

````
Step 1/9 : FROM golang:1.11-alpine as builder
 ---> e116d2efa2ab
Step 2/9 : RUN apk add --no-cache make gcc musl-dev linux-headers bash cmake g++
 ---> Using cache
 ---> 4ab58d60f4d0
Step 3/9 : ADD . /PlatON-Go
 ---> Using cache
 ---> c67eb4aaf944
Step 4/9 : RUN cd /PlatON-Go && make platon
 ---> Running in 2845f08df29e
build/build_deps.sh
make[1]: Entering directory '/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls'
g++ -g3 -Wall -Wextra -Wformat=2 -Wcast-qual -Wcast-align -Wwrite-strings -Wfloat-equal -Wpointer-arith -m64 -I include -I test -fomit-frame-pointer -DNDEBUG -Ofast  -fPIC -std=c++11 -I/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include -c src/bls_c256.cpp -o obj/bls_c256.o -MMD -MP -MF obj/bls_c256.d
g++ -g3 -Wall -Wextra -Wformat=2 -Wcast-qual -Wcast-align -Wwrite-strings -Wfloat-equal -Wpointer-arith -m64 -I include -I test -fomit-frame-pointer -DNDEBUG -Ofast  -fPIC -std=c++11 -I/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include -c src/bls_c384.cpp -o obj/bls_c384.o -MMD -MP -MF obj/bls_c384.d
g++ -g3 -Wall -Wextra -Wformat=2 -Wcast-qual -Wcast-align -Wwrite-strings -Wfloat-equal -Wpointer-arith -m64 -I include -I test -fomit-frame-pointer -DNDEBUG -Ofast  -fPIC -std=c++11 -I/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include -c src/bls_c384_256.cpp -o obj/bls_c384_256.o -MMD -MP -MF obj/bls_c384_256.d
g++ -g3 -Wall -Wextra -Wformat=2 -Wcast-qual -Wcast-align -Wwrite-strings -Wfloat-equal -Wpointer-arith -m64 -I include -I test -fomit-frame-pointer -DNDEBUG -Ofast  -fPIC -std=c++11 -I/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include -c src/bls_c512.cpp -o obj/bls_c512.o -MMD -MP -MF obj/bls_c512.d
In file included from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/op.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/fp.hpp:29,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/fp_tower.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/bn.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/bn256.hpp:10,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/impl/bn_c_impl.hpp:9,
                 from src/bls_c_impl.hpp:7,
                 from src/bls_c256.cpp:2:
/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/gmp_util.hpp:37:10: fatal error: gmpxx.h: No such file or directory
 #include <gmpxx.h>
          ^~~~~~~~~
In file included from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/op.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/fp.hpp:29,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/fp_tower.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/bn.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/bn384.hpp:10,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/impl/bn_c_impl.hpp:11,
                 from src/bls_c_impl.hpp:7,
                 from src/bls_c384.cpp:2:
/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/gmp_util.hpp:37:10: fatal error: gmpxx.h: No such file or directory
 #include <gmpxx.h>
          ^~~~~~~~~
compilation terminated.
compilation terminated.
make[1]: *** [Makefile:82: obj/bls_c384.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [Makefile:82: obj/bls_c256.o] Error 1
In file included from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/op.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/fp.hpp:29,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/fp_tower.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/bn.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/bls12_381.hpp:11,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/impl/bn_c_impl.hpp:13,
                 from src/bls_c_impl.hpp:7,
                 from src/bls_c384_256.cpp:3:
/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/gmp_util.hpp:37:10: fatal error: gmpxx.h: No such file or directory
 #include <gmpxx.h>
          ^~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:82: obj/bls_c384_256.o] Error 1
In file included from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/op.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/fp.hpp:29,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/fp_tower.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/bn.hpp:9,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/bn512.hpp:10,
                 from /PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/impl/bn_c_impl.hpp:15,
                 from src/bls_c_impl.hpp:7,
                 from src/bls_c512.cpp:2:
/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls/../mcl/include/mcl/gmp_util.hpp:37:10: fatal error: gmpxx.h: No such file or directory
 #include <gmpxx.h>
          ^~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:82: obj/bls_c512.o] Error 1
make[1]: Leaving directory '/PlatON-Go/crypto/bls/bls_linux_darwin/src/bls'
make: *** [Makefile:17: platon] Error 2
The command '/bin/sh -c cd /PlatON-Go && make platon' returned a non-zero code: 2
````

**Comment by benbaley** [comment]
The Dockerfile has some problems and we will fix it later.

**Comment by benbaley** [comment]
fixed in #1729

**Comment by benbaley** [comment]
Closed due to prolonged lack of feedback
